scripts/tmux-user-shell.sh

```diff
@@ -31,7 +32,8 @@ echo
 echo "This shell provides the following aliases:"
 echo ""
 echo "  fedimint-cli   - cli client to interact with the federation"
-echo "  ln1, ln2       - cli clients for the two lightning nodes (1 is gateway)"
+echo "  lightning-cli  - cli client for Core Lightning"
+echo "  lncli          - cli client for LND"
```

Comment:
docs: which one is the gateway?

Reply:
How will users choose which gateway to use in that future? We might want to add the ability to name gateways.

Reply:
To get the tests passing I defined these https://github.com/fedimint/fedimint/pull/1719/files#diff-5838719667e50a69f27dfddac131cad8ed4e11bee597a64a1b48454efdfad40bR167-R187

But yea naming them might be interesting, too.